test/support/api/clinicians.js

```diff
@@ -55,7 +55,8 @@ Cypress.Commands.add('routeClinicians', (mutator = _.identity) => {
   cy.route({
     url: '/api/clinicians',
     response() {
-      const clinicians = getResource(fxClinicians, 'clinicians');
+      // Remove current clinician
```

Comment:
@nmajor25 this was the major issue with this stub flake.  Fixing this let me rollback a few of the other changes.  See what you think.

Reply:
Nice, looks good 👍 And that test appears to be running smoothly now, without flake.